library/sha1.c

```diff
@@ -71,8 +71,15 @@
 }
 #endif
 
+#define MBEDTLS_SHA1_VALIDATE_RET(cond)                             \
```

Comment:
Formatting of these macros is inconsistent

Reply:
Fixed.